package.json

```diff
@@ -169,13 +169,14 @@
     "rollup": "^1.16.2",
     "rollup-plugin-babel": "^4.3.3",
     "rollup-plugin-terser": "^5.0.0",
-    "rollup-plugin-typescript2": "^0.21.2",
+    "rollup-plugin-typescript2": "^0.29.0",
     "shelljs": "^0.8.3",
     "start-server-and-test": "^1.11.5",
     "storybook-readme": "^5.0.5",
     "terser": "^5.3.0",
-    "typedoc": "^0.16.9",
-    "typescript": "3.7.5",
+    "typedoc": "^0.20.12",
+    "typescript": "4.1.3",
+    "tslib": "^2.0.3",
```

Comment:
Needed to make `rollup-plugin-typescript2` work: https://github.com/yarnpkg/berry/issues/2140